dbt/seeds/model/schema.yml

```diff
@@ -0,0 +1,47 @@
+seeds:
+  - name: model.final_model
```

Comment:
**[Thought, non-blocking]** What do we think about adding some tests to this model? In particular, I'm thinking about how the join conditions in this PR depend on the assumption that the combination of `township_code_coverage` values for final models in a given year cover all triads in that year's township. I'm not 100% confident that our current strategy for testing changed and added models would work for seeds (we know that models will get tested if the queries that comprise them change, but is the same for seeds whose data change?) but if we could get it working I think it would be a helpful safeguard for future maintainers who will need to modify the `model.final_model.csv` seed file.

Just an idea for the future, we can do this investigation in a follow-up PR!

Reply:
Let's kick this to a follow-up issue. See #376